ansys/dpf/core/plotter.py

```diff
@@ -770,7 +749,7 @@ def plot_contour(
             version_to_reach = '0.32.0'
```

Comment:
Hi @PProfizi, could you please check that it does not break back the following : https://github.com/pyansys/pydpf-core/issues/140 ?

Reply:
@anslpa I did, I found a bug, I fixed it, it should work :)